index.html

```diff
@@ -0,0 +1,203 @@
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>
+
+
+        <div class="card4">
+            <img src="image/card4.png" alt="">
+        <h2>Multi-Post</h2>
+        <h3>Stories</h3> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="image/image4.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>
+    
+    </section>
+
+    <section>
+    
+    <div class="about">
+        <h2>About </h2>
+        <h3> MySelf</h3>
+        <p class="intro">
+            Hello I am a software developer! I can help you build a product ,feature or website Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <div class="contact">
+            <h4>LET'S  CONNECT</h4>
+             </div>
+             <div class="icons">
+        <ul>
+          <li><img src="image/image6.png" alt="twitter"></li>
+          <li><img src="image/image7.png" alt="twitter"></li>
+          <li><img src="image/image8.png" alt="twitter"></li>
+           <li><img src="image/image9.png" alt="twitter"></li>
+          <li><img src="image/imge5.png" alt="twitter"></li>
+        </ul>
+        </div>
+          
+          <button type="button">Get My Resume</button>
+
+                
+                <div class="end">
+                 <ul class="photo">
+                <li><h5>Languages</h5><img src="/image/imge8.png" alt="photo"></li>
+                </ul>
+                        <div class="fitems">
+                             <ul>
+                            <li><img src="/image/Ellipse1.png" alt="1">Javascript</li>
+                           </ul>
+                            <ul>
+                            <li><img src="/image/Ellipse2.png" alt="2">CSS</li>
+                            </ul>
+                             <ul>
+                            <li><img src="/image/ellipse3.png" alt="3">HTML</li>
+                            </ul>
+                          </div>
+                        
+            
+            <ul class="photo1">
+                <li><h5>Frameworks</h5><img src="/image/imge9.png" alt="photo"></li>
+            </ul>
+            <ul>
+                <li><img src="/image/imgg1.png" alt="line"></li>
+            </ul>
```

Comment:
- [ ] [OPTIONAL]Please note that the images in this section are not showing. Fix  the path to this images so that your project will be complete

This is how your project currently appears
<img width="363" alt="Screenshot 2022-09-24 at 07 50 30" src="https://user-images.githubusercontent.com/48546992/192080590-bdfbb03c-affc-4d62-8c8e-e6b176026462.png">

- [ ] Also Kindly fix the margin at the bottom. Note that after skills the container has a margin bottom
this is the templates design.
<img width="192" alt="Screenshot 2022-09-24 at 07 55 24" src="https://user-images.githubusercontent.com/48546992/192080688-ebc29ded-0b78-4b38-b759-9eabe5ee0ff4.png">